src/components/errorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  onNotificationClose: () => void,
+  isErrorOccuring: boolean,
+};
+export const ErrorNotification: React.FC<Props> = ({
+  onNotificationClose,
+  isErrorOccuring,
+}) => {
+  const onCloseHandler = () => {
+    onNotificationClose();
+  };
```

Comment:
Why do you need to wrap this function into another function?

Reply:
Because I call it on click event, If I dont, I`ll get this 
![image](https://user-images.githubusercontent.com/96202644/222411922-f870b61f-d87d-4a17-941b-a2eda55f7bf1.png)

Reply:
I've added the comment below